doc/meetings/build-wg-meeting-2017-12-05.md

```diff
@@ -0,0 +1,121 @@
+# Node.js Foundation Build WorkGroup Meeting 2017-12-05
+
+## Links
+
+* **Recording**:  https://www.youtube.com/watch?v=6TT90qgE-pU
+* **GitHub Issue**: https://github.com/nodejs/build/issues/1028
+
+## Present
+
+* Michael Dawson (@mhdawson)
+* Jon Moss (@maclover7)
+* Refael Ackermann (@refack)
+* Kyle Farnung (@kfarnung)
+* Rod Vagg (@rvagg)
+* George Adams (@gdams)
+* Rich Trott (@Trott)
+* Gibson Fahnestock (@gibfahn)
+* João Reis (@joaocgreis)
+
+## Standup
+
+* George Adams (@gdams)
+  * Working on macOS Ansible scripts
+  * Started looking at ppcle ansible
+* João Reis (@joaocgreis)
+  * Created VS2017 release machines, will move forward with test CI.
+* Jon Moss (@maclover7)
+  * Status update job is now done, playing with pipelines
+* Kyle Farnung (@kfarnung)
+  * Just got back, started looking at docker
+* Michael Dawson (@mhdawson)
+  * working on macOS infra, job to test out new machines
+  * working on addition of z/OS machines and libuv jobs (in and green)
+  * working to fixup put compilers on PPC machines
+  * working to get/setup additional PCC/zLinux test machines to support compiler wok
+  * some work on process to recover when we don’t have ArmV6 in release
+* Refael Ackermann (@refack)
+  * Maintenance
+  * CI Job tweaks
+  * core build scripts (Makefile, .gyp, and vcbuild)
+  * GCC upgrade investigation
+* Rich Trott (@Trott)
+* Rod Vagg (@rvagg)
+* Gibson Fahnestock (@gibfahn)
+  * looked at OSX ansible stuff with George
+  * working on job to test cmake replacement for gyp
+
+## Agenda
+
+* New OSX infra walkthrough [#1026](https://github.com/nodejs/build/issues/1026)
+* Provide read access to our ci.nodejs.org configs [#972](https://github.com/nodejs/build/issues/972)
+* Ubuntu non-LTS strategy [#967](https://github.com/nodejs/build/issues/967)
+* Remove ARM64 Ubuntu 14.04 X-Gene servers from CI [#966](https://github.com/nodejs/build/issues/966)
+* Remove Fedora 22, 23, 24 [#962](https://github.com/nodejs/build/issues/962)
+* Remove Ubuntu 12.04 [#961](https://github.com/nodejs/build/issues/961)
+* Make it easier for people to join the Build WG [#941](https://github.com/nodejs/build/issues/941)
+* Permission to start culling all pre-v4 infra and build logic [#926](https://github.com/nodejs/build/issues/926)
+* suggestion: investigate a commit-queue solution [#705](https://github.com/nodejs/build/issues/705)
+* [#1030] https://github.com/nodejs/build/issues/1030
+* Blue overview
+
```

Comment:
add
  Jon gave us an overview of the Jenkins Blue UI and some work he's done on pipelines so far.